azure-pipelines.yml

```diff
@@ -161,24 +209,27 @@ stages:
                 feedsToUse: 'select'
                 includeNuGetOrg: true
                 arguments: 'install --global CycloneDX'
+
             - task: DotNetCoreCLI@2
               displayName: Dotnet generate SBOM
               inputs:
                 command: custom
                 custom: CycloneDX
                 projects: '$(solution)'
                 arguments: '-t -j -o build'
+
             - task: DotNetCoreCLI@2
               env:
-                SignAssembly: 'false'  # ToDo: Code signing is temporarily disabled until new certificate is properly set up
-              condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
+                SignAssembly: $(IS_RELEASE_BRANCH)
```

Comment:
In:
- https://github.com/SonarSource/sonar-scanner-msbuild/blob/master/src/Directory.Build.targets#L27
- https://github.com/SonarSource/sonar-scanner-msbuild/blob/master/src/SonarScanner.MSBuild.TFS/Properties/InternalsVisibleTo.cs#L23
- https://github.com/SonarSource/sonar-scanner-msbuild/blob/master/src/SonarScanner.MSBuild.Shim/Properties/InternalsVisibleTo.cs#L23
- ...

Reply:
The preprocessor symbol could also be specified on the dotnet command itself instead of the indirection with the directory targets file. You may want to change this, ignore my comment, or create an issue.